apps/builder/app/builder/features/inspector/inspector.tsx

```diff
@@ -68,6 +68,29 @@ export const Inspector = ({ publish, navigatorLayout }: InspectorProps) => {
     );
   }
 
+  if (selectedInstance.component === "Slot") {
```

Comment:
can we please just not show the tab triggers where they are right now instead of copy pasting entire block?
we may have other conditions in the future, so showStylePanel and showPropsPanel would be nice

Reply:
Default is also different. Though most of the time is not correct because tab remembers style.

Reply:
wait we don't want to remove it, just disable it

Reply:
there is even disabled prop on trigger from radix